lms/views/basic_lti_launch.py

```diff
@@ -55,6 +56,10 @@ def __init__(self, context, request):
         # Configure the front-end mini-app to run.
         self.context.js_config.update({"mode": "basic-lti-launch"})
 
+        # Configure front-end grading if feature is enabled
+        if request.feature("blackboard_grading"):
+            frontend_app.configure_grading(request, self.context.js_config)
```

Comment:
The idea is that we could move more of the `js_config` futzing into this `frontend_app` helper module to get the logic out of the views themselves.